libs/designer-ui/src/lib/editor/base/utils/parsesegments.ts

```diff
@@ -43,7 +43,7 @@ export const parseHtmlSegments = (value: ValueSegment[], tokensEnabled?: boolean
           data: segment,
           brandColor,
           icon: icon,
-          value: value ?? segmentValue,
+          value: segmentValue ?? value,
         });
```

Comment:
Didn't you already have this change made in some PR

Reply:
I reversed this actually because it was showing the wrong value ... but I'm not able to repro the wrong value any more... so I'm just going to revert it

Reply:
> Didn't you already have this change made in some PR

I'm unable to find this change in the change log, maybe it never got merged?

Reply:
oh actually I found why, its in this PR https://github.com/Azure/LogicAppsUX/pull/2406/files, which hasn't checked in